programming/autonomous.md

```diff
@@ -0,0 +1,41 @@
+# Creating an Autonomous
+
+Autonomous is code that is used to make the robot move by itself. 
+Our autonomous mainly uses a Boilerplate to create states for the robot to follow.
+We utilize the ```AutonomousStateMachine``` from ```magicbot``` to create a progression of commands for the robot to follow.
+Create your autonomous file in a directory called "Autonomous" in your project.
```

Comment:
Autonomous should be lowercase and should have single backticks rather than quotes